src/test/java/LibraryTest.java

```diff
@@ -371,4 +371,13 @@ public void testGetAllMethods() {
     assertNotEquals("multiArrayConcat", list.get(0));
     assertNotEquals("arrayConcat", list.get(1));
   }
+
+  /**
+   * Tests for {@link Library#getAllFieldNames(Class)}.
+   */
+  @Test
+  public void testGetAllFieldNames() {
+    var list = Library.getAllFieldNames(Library.class);
+    assertEquals(list.size(), 0);
```

Comment:
I think we should extend the test a bit so that it actually finds a public field. Maybe introduce another class for this? It could be inline to keep it short and clean.

Reply:
https://docs.oracle.com/javase/tutorial/java/javaOO/localclasses.html

Reply:
Addressed. Updated the test case